department-of-veterans-affairs/va.gov-cms
# VACMS-10989: Adds new user count metrics

## Description

Closes #10989.

I need to review with @stefaniefgray to ensure that my statistics match how she gathers hers.

## Testing done


## Screenshots


## QA steps
Visit `/metrics` and verify that the metrics now include something like the following:

```
drupal_va_gov_active_user_count_total 1
drupal_va_gov_active_user_count_total{last="session_write_interval"} 0
drupal_va_gov_active_user_count_total{last="60days"} 1
drupal_va_gov_active_user_count_total{last="180days"} 1
```

Note that the figures are extremely low.  This is because the rows are `DISTINCT` by user email, and because the CMS DB sanitization process removes user emails.  This will return a different result on production.
